scripts/travis-screenshots

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+# Decrypt the deployment key
+openssl aes-256-cbc -K $encrypted_1645300b04d0_key -iv $encrypted_1645300b04d0_iv -in .travis.sskey.enc -out .travis.sskey -d
+
+# Configure ssh access using the key
+mv .travis.sskey ~/.ssh/id_rsa_deploy
```

Comment:
Why do we need this key at all, when we didn't before? What's wrong with the token system?